docs/user/reuse_garage_policy.md

```diff
@@ -0,0 +1,54 @@
+# Load and Use a Trained Policy
+
+In this section you will learn how to save and load snapshots of your experiment
```

Comment:
Saving/loading snapshots is covered by a different tutorial. Please keep the focus here singular.